packages/app-extrinsics/src/Extrinsic.tsx

```diff
@@ -73,6 +75,10 @@ class Extrinsic extends React.PureComponent<Props, State> {
   }
 
   nextState (newState: State): void {
+    const isValidLength = (values: Array<RawParam>, index: number) => {
+      return (values[index].type === 'Balance' ? checkValueBitLength(values[index].value as BN) : true);
```

Comment:
This would read a bit better split,

```
const isValidLength = (values: Array<RawParam>, index: number) =>
      return values[index].type === 'Balance'
          ? checkValueBitLength(values[index].value as BN)
           : true;
```